integreat_cms/cms/forms/pages/page_form.py

```diff
@@ -181,10 +181,10 @@ def _clean_cleaned_data(self):
 
     def get_editor_queryset(self):
```

Comment:
Maybe this function should be renamed? Right now I find it a bit confusing since it does not actually return the page editors anymore